test/pkg/environment/expectations.go

```diff
@@ -125,6 +128,31 @@ func (env *Environment) eventuallyExpectScaleDown() {
 	}).Should(Succeed(), fmt.Sprintf("expected scale down to %d nodes, had %d", env.Monitor.NodeCountAtReset(), env.Monitor.NodeCount()))
 }
 
+func (env *Environment) GetCreatedNodes(before []v1.Node, after []v1.Node) []*v1.Node {
+	createdNodes := []*v1.Node{}
+	oldList := sets.NewSet()
```

Comment:
set.Without() might do this for you.

Reply:
speaking complexity wise: I'd still have to translate the `after` list into a set, and then do `set.Without(set)`, which is technically more expensive (although only by a teensy bit). I also don't see the `Without` function.